File: BAEKJOON/Javascript/5000/5500/5597.js
/*
Title : 과제 안 내신 분..?
Link : https://www.acmicpc.net/problem/5597
*/

const fs = require("fs");
const filepath =
  process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt";
const input = fs.readFileSync(filepath).toString().trim().split("\n");

const students = Array.from({ length: 30 }, () => 0);
input.forEach((student) => {
  const index = Number(student) - 1;
  students[index] = 1;
});

const notSubmitted = students.reduce((acc, cur, index) => {
  if (cur === 0) {
    acc.push(index + 1);
  }
  return acc;
}, []);

console.log(notSubmitted.join("\n"));

File: BAEKJOON/Python/5000/5500/5597.py
"""
Title : 과제 안 내신 분..?
Link : https://www.acmicpc.net/problem/5597
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    students = [False] * 31
    students[0] = True
    for _ in range(28):
        students[int(input().strip())] = True
    for i in range(1, 31):
        if not students[i]:
            print(i)
